scripts/auto-rebase/rebase.sh

```diff
@@ -385,7 +402,7 @@ EOF
          test -n "$(git status -s scripts/auto-rebase/last_rebase.sh)" && \
          title "## Committing changes to last_rebase.sh" && \
          git add scripts/auto-rebase/last_rebase.sh && \
-         git commit -m "update last_rebase.sh" || true)
+         git commit -m "update last_rebase.sh") || true
```

Comment:
This was error https://www.shellcheck.net/wiki/SC2015

Was the intention here `(this AND that) OR true`?

Reply:
Right, that should function as intended. Thanks!